pygfx/materials/_line.py

```diff
@@ -101,26 +124,41 @@ def map(self, map):
 
 
 class LineThinMaterial(LineMaterial):
-    """A simple line, drawn with line_strip primitives that has a thickness
+    """Thin line material.
+
+    A simple line, drawn with line_strip primitives that has a thickness
     of one physical pixel (the thickness property is ignored).
 
     You probably want to avoid this material, because its width is
     inconsistent and looks *very* thin on HiDPI monitors.
     """
 
+    # @almarklein: The docs advise to avoid this. Depricate before 1.0?
+
```

Comment:
It may still be useful because the thin lines are more performant, or for e.g. debugging, so I'm reluctant to remove them.

Reply:
No problem. In that case, I'd say we just add that info to the docs so it is clear why it exists.